### PR TITLE
mu search page person

### PR DESCRIPTION
**Pull request by nbittich**

### CODE REVIEW

**Review by Windvis — COMMENTED** [review]
Looks good to me and the frontend seemed to work :+1:.

@claire-lovisa is probably a better person to review this though :smile:.

File: docker-compose.yml [hunk 1]
```diff
@@ -332,3 +332,27 @@ services:
...
+      image: nbittich/mu-search:wip
+      links:
+        - db:database
+      volumes:
+        - ./config/search:/config
+    # environment:
+          # LOG_SCOPE_DELTA: "debug"
+          # LOG_SCOPE_UPDATE_HANDLER: "debug"
+          # LOG_SCOPE_SEARCH: "debug"
+          # LOG_SCOPE_ELASTICSEARCH: "info"
```
Comment (claire-lovisa) [inline docker-compose.yml:350]: We could probably remove this ? To keep a docker-compose file as small as possible, I know it tends to grow fast already :sweat_smile:

**Review by claire-lovisa — COMMENTED** [review]
Not the expert in mu-search haha, looks good anyways !

**Review by claire-lovisa — APPROVED** [review]
Tested and approved ! :)

**Pull request merged**